src/transformers/models/bart/configuration_bart.py

```diff
@@ -168,7 +171,12 @@ def __init__(
         self.num_hidden_layers = encoder_layers
         self.gradient_checkpointing = gradient_checkpointing
         self.scale_embedding = scale_embedding  # scale factor will be sqrt(d_model) if True
-        self.force_bos_token_to_be_generated = force_bos_token_to_be_generated  # only relevant for CNN
+
+        # ensure backward compatibilty for BART CNN models
+        self.forced_bos_token_id = (
+            bos_token_id if kwargs.get("force_bos_token_to_be_generated", False) else forced_bos_token_id
+        )
+        self.forced_eos_token_id = forced_eos_token_id if forced_eos_token_id is None else eos_token_id
```

Comment:
We don't need this for backwards compatibility I think -> After this PR `forced_eos_token_id` will always be 2 which leads to exactly the same behavior as before

Reply:
think we should delete this line